From 048073cc9b94da56af36afbb362a64be32a2d846 Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@suse.de>
Date: Thu, 4 Dec 2008 20:39:37 -0800
Subject: Staging: rtl8187se: remove duplicate pci ids

We only want to load for one PCI device, the "SE" device.  The other PCI
devices that this driver supports are already supported by the existing
rtl8187 module.

Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/rtl8187se/r8180_core.c |    2 ++
 1 file changed, 2 insertions(+)

--- a/drivers/staging/rtl8187se/r8180_core.c
+++ b/drivers/staging/rtl8187se/r8180_core.c
@@ -102,6 +102,7 @@ static struct pci_device_id rtl8180_pci_
                 .subdevice = PCI_ANY_ID,
                 .driver_data = 0,
         },
+#if 0
         {
                 .vendor = PCI_VENDOR_ID_BELKIN,
                 .device = 0x6001,
@@ -130,6 +131,7 @@ static struct pci_device_id rtl8180_pci_
 		.subdevice = PCI_ANY_ID,
 		.driver_data = 4,
 	},
+#endif
         {
                 .vendor = 0,
                 .device = 0,
